src/spec.c

```diff
@@ -1148,20 +1163,35 @@ void IndexSpec_FreeSync(IndexSpec *spec) {
 
 //---------------------------------------------------------------------------------------------
 
-void Indexes_Free(dict *d) {
-  arrayof(IndexSpec *) specs = array_new(IndexSpec *, 10);
+void Indexes_Free(dict *d, TrieMap *schemaPrefixes, void *aliases, void *cursors) {
```

Comment:
Why void*?

Reply:
Due to circular include of header files.